dns/bind/src/opnsense/mvc/app/models/OPNsense/Bind/Record.xml

```diff
@@ -0,0 +1,41 @@
+<model>
+    <mount>//OPNsense/bind/record</mount>
+    <description>BIND record configuration</description>
+    <version>1.0.0</version>
+    <items>
+        <records>
+            <record type="ArrayField">
+                <enabled type="BooleanField">
+                    <default>1</default>
+                    <Required>Y</Required>
+                </enabled>
+                <domain type="ModelRelationField">
+                    <Required>Y</Required>
+                    <Model>
+                        <template>
+                            <source>OPNsense.Bind.Domain</source>
+                            <items>domains.domain</items>
+                            <display>domainname</display>
+                        </template>
+                    </Model>   
+                </domain>
+                <name type="TextField">
+                    <default></default>
+                    <Required>N</Required>
+                </name>
+                <type type="OptionField">
+                    <default>A</default>
+                    <Required>Y</Required>
+                    <OptionValues>
+                        <A>A</A>
+                        <CNAME>CNAME</CNAME>
+                        <MX>MX</MX>
```

Comment:
you will not get far without SRV-, TXT and CAA records. I would also recommend to support NS records.

Reply:
As I said ... please don't review, there are tons of features missing. Only opened to get some help regarding templating :)